dist/js/bootstrap.js

```diff
@@ -8,9 +8,9 @@ if (typeof jQuery === 'undefined') {
   throw new Error('Bootstrap\'s JavaScript requires jQuery')
 }
 
-+function ($) {
+function ($) {
   'use strict';
-  var version = $.fn.jquery.split(' ')[0].split('.')
+  var version = $.fn.jquery.split(' ')[0].split('.');
```

Comment:
Hmmm ... my editor missed master/js/.jshintrc I guess. Sorry for bothering.

Thanks ++ voor de replies though :-)